index.html

```diff
@@ -129,15 +137,59 @@
         <h1 data-h1="Maintenance">Maintenance</h1>
         <p data-p="Erreur sur la page">Erreur sur la page</p>
         <p>&nbsp;</p>
-        <p class="message">Nos équipes ont été notifiées et interviendront dans les meilleurs délais.</p>
+        <p class="message">
+          Nos équipes ont été notifiées et interviendront dans les meilleurs
+          délais.
+        </p>
       </div>
     </div>
     <footer>
-      Fabrique numérique des Ministères Sociaux - <a href="mailto:DSI-incubateur@sg.social.gouv.fr">DSI-incubateur@sg.social.gouv.fr</a>
+      Fabrique numérique des Ministères Sociaux -
+      <a href="mailto:DSI-incubateur@sg.social.gouv.fr"
+        >DSI-incubateur@sg.social.gouv.fr</a
+      >
     </footer>
     <script>
-      Raven.config('https://7b80ba66958845f4b44790ffc7191638@sentry.fabrique.social.gouv.fr/28').install();
-      Raven.captureException(new Error());
+      (function () {
+        var url = new URL(location);
```

Comment:
@revolunet [IE11 does not support `URL` apparently :(](https://caniuse.com/#feat=url)

Reply:
haha best IE11 joke ever :)

good to know !

Reply:
document.location should do the trick ?

Reply:
i can confirm it does `document.location.hostname` is available in IE11

Reply:
Change done (not tested)